rpcs3/Emu/RSX/RSXThread.cpp

```diff
@@ -654,7 +654,7 @@ namespace rsx
 #else
 			constexpr u32 host_min_quantum = 500;
 #endif
-			u64 start_time = rsx::uclock();
+			u64 start_time = get_system_time();
```

Comment:
Don't call this in a loop. I cannot stress enough how badly we're handling timers. There is a reason I moved to tsc timing. Use tsc and reconcile to real time if you really must when the loop is done.
I doubt you'll just take my word on this so I'll provide profiling traces with ntdll symbols when I am at my PC.

Reply:
get_guest_system_time is currently must only take the return values of get_system_time() for it to be accurate. And this is the VBLANK thread code, it runs this code once in about 16ms which is extremely insignificant.

Reply:
What get_guest_system_time does here is to convert host time to guest time which are entirely different from each other.

Reply:
What happens when using very high vblank timing like the 1800+ hz used by the mgs community?

Reply:
But there is no replacement, and it is quite flawed to use 1800hz in the first place.